src/styles/variables.js

```diff
@@ -21,11 +21,11 @@ const colors = {
 };
 
 const margin = {
-  gutter: "10px",
+  gutter: "1rem",
 };
 
 const padding = {
-  sides: "30px",
+  sides: "3rem",
```

Comment:
I don't think changing the global margin and padding is within scope. This is managed with media tags and meant for spacing and not font-sizes.